### PR TITLE
doc: add minutes for January 5th, 2017

### PR DESCRIPTION
**Pull request by sam-github**
Fix: https://github.com/nodejs/security-wg/issues/13

### CODE REVIEW

File: meetings/2017-01-05/minutes.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  require() would need to support loading js from the webpack file
+- Bradley: would need to specify trusted authorities who could have signed the
+  webpack
+- Bradley: would like to see security analysis of the webpack format
+- Sam: reasonable, the W3C is inventing a new encrypted file format, getting
+  vuln researchers to look at it critically would be a good idea. cf.
+  W3C: https://w3ctag.github.io/packaging-on-the-web/
+- James: suggests that an EPS should be opened and discussion should be there
+- Michael: agrees
+- XXX Link?
```
Comment (sam-github) [inline meetings/2017-01-05/minutes.md]: @bmeck I thought I saw a EPS issue for this, but I can't find it now, so maybe I mistook one of your other EPSs for this one.

**Comment by sam-github** [comment]
/to attendees: @cjihrig @jasnell @bmeck @vdeturckheim @gibfahn @mhdawson

File: meetings/2017-01-05/minutes.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
+# Security WG meeting 2017-01-05
+
+GitHub issue: https://github.com/nodejs/security-wg/issues/13
```
Comment (gibfahn) [inline meetings/2017-01-05/minutes.md]: These are all showing up on one line (cause there isn't a blank line between them). Best fix is probably to use bullet points.

File: meetings/2017-01-05/minutes.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+Meeting video: (not available, sorry, because of technical difficulties)
+Previous meeting: https://github.com/nodejs/security-wg/pull/15
+
+
+# Present
+
+* Sam Roberts
```
Comment (gibfahn) [inline meetings/2017-01-05/minutes.md]: Worth adding Github usernames.

File: meetings/2017-01-05/minutes.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
+# Security WG meeting 2017-01-05
+
+GitHub issue: https://github.com/nodejs/security-wg/issues/13
+Meeting video: (not available, sorry, because of technical difficulties)
+Previous meeting: https://github.com/nodejs/security-wg/pull/15
+
+
+# Present
```
Comment (gibfahn) [inline meetings/2017-01-05/minutes.md]: Should probably be `##` (everything should have one more `#` except the 1st line)

File: meetings/2017-01-05/minutes.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  - James: working on the process/policy of vulnerability reports is different
+    from fixing the bugs, so the process management could be TSC, and the fixing
+    of the vulns would remain with CTC
+  - Sam: anyone else have an opinion?
+  - Vladimir: neutral
+  - Sam: see https://github.com/nodejs/TSC/issues/175
+
+- James: does anyone want to take a stab at proposing a WG mandate?
+  - Sam: will open a PR against sec-wg, and put proposed charter in the README.md
+  - TBD
```
Comment (gibfahn) [inline meetings/2017-01-05/minutes.md]: Is there anything still TBD?

**Comment by sam-github** [comment]
@gibfahn PTAL

**Pull request merged**